atomic_reactor/plugins/post_koji_upload.py

```diff
@@ -308,10 +309,11 @@ def get_image_output(self):
 
     def get_digests(self):
         """
-        Returns a map of images to their digests
+        Returns a map of images to their digests and map of media types to their images
```

Comment:
How about:
`Returns a map of images to their digests and map of media types to each digest`

Reply:
Fixed.
Also consolidated the functions and wrote a little bit more of an explanation why.